tests/countries/test_faroe_islands.py

```
@@ -0,0 +1,53 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)